crates/solvers/src/infra/config/baseline.rs

```diff
@@ -46,7 +46,7 @@ struct Config {
 
     /// The amount of the native token to use to estimate native price of a
     /// token
-    native_token_price_estimation_amount: eth::U256,
+    native_token_price_estimation_amount: i64,
```

Comment:
Why not use `eth::U256` here? If you ran into the hex deserialization issue, this can be resolved with a `serde_as` macro.
But even if you decide to use built-in types it should definitely be unsigned since negative amounts don't make sense here.

Reply:
Ah forgot to clean this up. Fixed.